source/client-dependencies.txt

```diff
@@ -42,23 +286,72 @@ MongoDB instance.
 .. _authentication-options:
 
 Authentication Options
-----------------------
+~~~~~~~~~~~~~~~~~~~~~~
 
 .. include:: /includes/auth-options.rst
 
-.. example::
+Example
+~~~~~~~
+
+A |bi-short| user, ``grace``, authenticates with her username and
+password. For Challenge and Response, |ldap|, and Kerberos, the
+following examples show:
+
+- How to format the username to authenticate
+- Which configuration file settings :binary:`~bin.mongosqld`
+  requires to enable authentication
+
+.. tabs-auth::
+
+   tabs:
+     - id: uidpwd
+       content: |
+
+         .. example::
+
+            To authenticate as user ``grace`` using the ``admin``
+            database and the Challenge and
+            Response (``SCRAM-SHA-1``) authentication mechanism, write 
+            the username in this format:
+
+            .. code-block:: shell
+
+               grace?source=admin
+
+     - id: ldap
+       content: |
+
+         .. example::
+
+            To authenticate as user ``grace`` with the |ldap| 
+            (``PLAIN``) authentication mechanism, write the 
+            username in this format:
+
+            .. code-block:: shell
```

Comment:
This example, and the Kerberos one below, are colorizing oddly. I'd use ..code-block:: none to prevent colorization.